app/code/community/Ebizmarts/MailChimp/sql/mailchimp_setup/mysql4-upgrade-1.1.12-1.1.13.php

```diff
@@ -60,4 +60,7 @@
 ;
 $attribute->save();
 
+Mage::helper('mailchimp')->saveMailChimpConfig(array(array(Ebizmarts_MailChimp_Model_Config::ENABLE_POPUP, 0)), 0, 'default');
```

Comment:
magic numbers, change `0` to var or constant.
also, I think `default` can be read from a magento constant in a core class

Reply:
Thanks, I changed the way the pop is disabled, I realized using 'default' doesn't take into consideration if the config exists in the website or store scopes, now it deletes that configuration for all scopes.